lib/ae_mdw/db/oracle.ex

```diff
@@ -163,4 +162,20 @@ defmodule AeMdw.Db.Oracle do
         []
     end
   end
+
+  @spec list_expired_at(Blocks.height()) :: [pubkey()]
+  def list_expired_at(height) do
+    do_list_expired_at([], height, <<>>)
+  end
+
+  defp do_list_expired_at(oracle_pks, height, prev_pubkey) do
+    # includes revoked
+    case Database.next_key(Model.InactiveOracleExpiration, {height, prev_pubkey}) do
+      {:ok, {^height, oracle_pk}} ->
+        do_list_expired_at([oracle_pk | oracle_pks], height, oracle_pk)
+
+      _other_height ->
+        oracle_pks
+    end
+  end
```

Comment:
Can't we use the same function here as well?

Reply:
well reminded, thanks